bot/Dockerfile

```diff
@@ -1,32 +1,30 @@
-FROM python:3.7@sha256:6eaf19442c358afc24834a6b17a3728a45c129de7703d8583392a138ecbdb092
+FROM python:3.7-slim@sha256:e720c669cd7188d6a7e36e567be6bde9feb931d66219f1e4ba2caeb30e35d2cd
```

Comment:
looks good, one minor thing is we should probably have the tests run in a python:3.7-slim container instead of the current python:3.7 to head off any differences before release 

Also @chdsbd probably wants to run these containers on staging before merge although I don't think we use any OS specific stuff

Reply:
good point, I just updated the container to use slim version for the bot test step in the circleci config